engineio/async/eventlet.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
```

Comment:
I believe, yes, because 2.7 still has relative ones by default. Without this test project fails with ImportErrors for me, trying to import non-existent `engineio.async.gevent.queue` when it sees `import gevent.queue`.